src/docs/options.md

```diff
@@ -55,58 +55,87 @@ levels in different tasks.
 
+    :::toml
+    foo = 5
+    bar = 4.5
+
 ### String Options
 
-Surround strings with single or double quotes if they contain embedded spaces: `--foo="hello, world"`.
+Surround strings with single or double quotes: `--foo="hello, world"`.
+
+    :::toml
+    foo = "hello world"
 
 ### List Options
 
-List options can be appended to and filtered, as well as overridden.
-For example, for an option `--foo` whose default value is `[1, 2]`, then in `pants.ini`:
+List options can be added to and removed, as well as overridden.
+For example, for an option `--foo` whose default value is `[1, 2]`, then in `pants.toml`:
+
++ `foo = 3` will yield `[1, 2, 3]`.
++ `foo.add = [3, 4]` will yield `[1, 2, 3, 4]`.
++ `foo.remove = [1]` will yield `[2]`.
++ `foo = [3, 4]` will yield `[3, 4]`.
+
+In `pants.toml`, you may both add and remove simultaneously:
+
+    :::toml
+    # This will yield [2, 3, 4]
+    foo.add = [3, 4]
+    foo.remove = [1]
 
-+ `foo: 3` will yield `[1, 2, 3]`.
-+ `foo: +[3, 4]` will yield `[1, 2, 3, 4]`.
-+ `foo: -[1]` will yield `[2]`.
-+ `foo: [3, 4]` will yield `[3, 4]`.
+Adding a single value and replacing a list works the same as `pants.toml` for the command line and environment variables.
 
-Multiple append and filter expressions may be delimited with commas,
-allowing you to append and filter simultaneously:
++ `--foo=3` will yield `[1, 2, 3]`.
++ `--foo='[3, 4]'` will yield `[3, 4]`.
 
-+ `foo: +[3,4],-[1]` will yield `[2, 3, 4]`.
+The syntax is a little different from `pants.toml`, however, when removing values or adding multiple values.
 
-On the command line you can append single values multiple times:
++ `--foo='+[3, 4]'` will yield `[1, 2, 3, 4]`.
++ `--foo='-[1]'` will yield `[2]`.
++ `--foo='+[3,4],-[1]'` will yield `[2, 3, 4]`.
+
+On the command line, you can also add single values multiple times:
 
 + `--foo=3 --foo=4` will yield the value `[1, 2, 3, 4]`.
 
-Note that these command line values will be appended to the value determined from the defaults
-plus the values in `pants.ini`. To override the value, use `--foo=[3, 4]`.
+Note that these command line values will be added to the list determined from the defaults
+plus the values in `pants.toml`. To instead override the list, use `--foo='[3, 4]'`.
 
-Filters apply to the entire list constructed so far, and will filter all appearances of the value:
+Removes apply to the entire list constructed so far, and will remove all appearances of the value:
 
-+ `--foo=1 --foo=1 --foo=2 --foo=-[1]` will yield `[2, 2]`.
++ `--foo=1 --foo=1 --foo=2 --foo='-[1]'` will yield `[2, 2]`.
 
-Filters take precedence over appends, so you cannot "add something back in":
+Removes take precedence over adds, so you cannot "add something back in":
 
 + `--foo=-[2] --foo=2` will yield `[1]`.
 
 
 ### Dict Options
 
-Dict option values are Python-style dict literals: `--foo={"a":1,"b":2}`.
+Dict option values are Python-style dict literals: `--foo='{"a": 1,"b": 2}'`.
```

Comment:
... are Python-style dict literals: `--foo='{"a": 1,"b": 2}'` embedded in a string.

The the next new bit makes sense.

Reply:
Good suggestion. Thanks!